data/actionsmap/yunohost.yml

```diff
@@ -165,8 +165,11 @@ user:
                     full: --change-password
                     help: New password to set
                     metavar: PASSWORD
+                    nargs: "?"
+                    const: 0
                     extra:
                         pattern: *pattern_password
+                        comment: good_practices_about_user_password
```

Comment:
This line created a bug: the advices are displayed even if no password is changed...
I don't know how to avoid this bug without removing this line :/
```
root@ynh:/ynh-dev# yunohost user update ljfa -f LOL
You are now about to define a new user password. The password should be at least 8 characters long—though it is good practice to use a longer password (i.e. a passphrase) and/or to a variation of characters (uppercase, lowercase, digits and special characters).
```

Reply:
(I believe it was already an issue priori to this PR so not really a regression)

Reply:
(I think the proper fix involve obscure tweaking of the moulinette ...)

Reply:
What do we do ? we remove the advice or we keep it on each update command event if there are no option change-password ?

Reply:
To me this is to be addressed in another PR ... in the meantime it's not really a huge deal ... this issue has been here for a while now and we can live with it ...

Reply:
ok so i set the ready to merge flag

Reply:
"Une fois n'est pas coutume", zamentur was in fact right, we should probably remove that line